spec/lib/formatter/generic-formatter.spec.js

```diff
@@ -0,0 +1,112 @@
+"use strict";
+
+describe("formatter/genericFormatter", () => {
+    var defaultTransform, genericFormatterAsync, resMock;
+
+
```

Comment:
We talked about the whitespace rules in person.  The short answer is yes, functions should be offset by two lines from anything near them.  Comments are stuck to the code below them, so the whitespace moves up above the comment.